lib/recurly/base.php

```diff
@@ -133,10 +142,22 @@ protected function __valuesString() {
     return implode($values, ', ');
   }
 
+  public function getHref() {
+    return $this->_href;
+  }
+  // TODO: not sure this should be public
+  public function setHref($href) {
```

Comment:
Yeah, I saw that you moved it. I'm more just curious as to how the one line that uses setHref works when the href should always be in the Location header